### PR TITLE
docs(README): add getting started section

### LINKED ISSUE #56: add a getting started guide

### PR DESCRIPTION
**Pull request by michelleN**
resolves #56 and #55

### CODE REVIEW

**Comment by michelleN** [comment]
Thanks for all the feedback @grampelberg @lachie83. This is ready for another round if reviews. :D

File: deploy/kubernetes-manifests.yaml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    metadata:
+      labels:
+        name: smi-adapter-istio
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      serviceAccountName: smi-adapter-istio
+      containers:
+        - name: smi-adapter-istio
+          image: {{ .Values.image.name }}
```
Comment (Pothulapati) [inline deploy/kubernetes-manifests.yaml]: This should be `deislabs/smi-adapter-istio:latest`, as we are using a single delpoyment file, right?

Comment (michelleN) [reply deploy/kubernetes-manifests.yaml]: fixed thank you!

**Pull request merged**